apps/i18n/common/en_us.json

```diff
@@ -27,6 +27,114 @@
   "directionSouthLetter": "S",
   "directionEastLetter": "E",
   "directionWestLetter": "W",
+  "dropletBlock_arcLeft_description": "Move the turtle in a counterclockwise arc using the specified number of degrees and radius",
+  "dropletBlock_arcRight_description": "Move the turtle in a clockwise arc using the specified number of degrees and radius",
+  "dropletBlock_button_description": "Create a button and assign it an element id",
+  "dropletBlock_checkbox_description": "Create a checkbox and assign it an element id",
+  "dropletBlock_circle_description": "Draw a circle on the active  canvas with the specified coordinates for center (x, y) and radius",
+  "dropletBlock_clearCanvas_description": "Clear all data on the active canvas",
+  "dropletBlock_clearInterval_description": "Clear an existing interval timer by passing in the value returned from setInterval()",
+  "dropletBlock_clearTimeout_description": "Clear an existing timer by passing in the value returned from setTimeout()",
+  "dropletBlock_console.log_description": "Log a message or variable to the output window",
+  "dropletBlock_container_description": "Create a division container with the specified element id, and optionally set its inner HTML",
+  "dropletBlock_createCanvas_description": "Create a canvas with the specified id, and optionally set width and height dimensions",
+  "dropletBlock_createRecord_description": "Creates a new record in the specified table.",
+  "dropletBlock_createRecord_param0": "table",
```

Comment:
I wonder if the parameter names will ever need to be localized? I don't think they will. If not, I'd rather see those names encoded in the dropletConfig objects, where the blocks are defined - right alongside the example parameter values.